code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -1811,3 +1811,137 @@
 				continue
 			L.adjustSanityLoss(-heal_amount)
 			new /obj/effect/temp_visual/healing(get_turf(L))
+
+/obj/item/ego_weapon/blind_obsession//When I saw that Ishmael's version was an anchor I thought "hey would it be funny if it was a throwing weapon with aoe".
+	name = "blind obsession"
+	desc = "All hands, full speed toward where the lights flicker. The waves... will lay waste to everything in our way."
+	special = "When thrown, this weapon attacks EVERYONE but yourself in an AOE. \
+			This weapon deals 75% more damage on direct throws."//Throw it at the Eo I dare ya!
+	icon_state = "blind_obsession"
+	force = 80
+	attack_speed = 2.5
+	throwforce = 100 //has aoe
+	throw_speed = 5
+	throw_range = 7
+	damtype = RED_DAMAGE
+	attack_verb_continuous = list("bashes", "smashes")
+	attack_verb_simple = list("bashes", "smashes")
+	hitsound = 'sound/weapons/ego/hammer.ogg'
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 80
+							)
+
+/obj/item/ego_weapon/blind_obsession/on_thrown(mob/living/carbon/user, atom/target)//No, clerks cannot hilariously kill others with this
+	if(!CanUseEgo(user))
+		return
+	return ..()
+
+/obj/item/ego_weapon/blind_obsession/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	playsound(src, 'sound/weapons/ego/hammer.ogg', 300, FALSE, 9)
+	var/damage = 75
+	if(ishuman(thrownby))
+		damage *= 1 + (get_modified_attribute_level(thrownby, JUSTICE_ATTRIBUTE))/100
+	damage *= force_multiplier
+	for(var/turf/open/T in range(2, src))
+		var/obj/effect/temp_visual/small_smoke/halfsecond/smonk = new(T)
+		smonk.color = COLOR_TEAL
+		if(!ismob(thrownby))
+			continue
+		thrownby.HurtInTurf(T, list(thrownby), damage, RED_DAMAGE)
+	return ..()
+
+/obj/item/ego_weapon/abyssal_route //An ungodly love child of sword sharpened with tears and fluid sac
+	name = "abyssal route"//old korean name I think
+	desc = "I am the only one who moves in these waves. ... Shatter."
+	special = "This weapon has a combo system ending with a dive attack. To turn off this combo system, use in hand. \
+			This weapon has a fast attack speed"
+	icon_state = "abyssal_route"
+	force = 20
+	damtype = BLACK_DAMAGE
+	attack_verb_continuous = list("stabs", "attacks", "slashes")
+	attack_verb_simple = list("stab", "attack", "slash")
+	hitsound = 'sound/weapons/ego/rapier1.ogg'
+	attribute_requirements = list(
+							JUSTICE_ATTRIBUTE = 80
+							)
+	var/combo = 0
+	var/combo_time
+	var/combo_wait = 10
+	var/combo_on = TRUE
+	var/can_attack = TRUE
+
+/obj/item/ego_weapon/abyssal_route/attack_self(mob/user)
+	..()
+	if(combo_on)
+		to_chat(user,"<span class='warning'>You swap your grip, and will no longer perform a dive finisher.</span>")
+		combo_on = FALSE
+		return
+	if(!combo_on)
+		to_chat(user,"<span class='warning'>You swap your grip, and will now perform a dive finisher.</span>")
```

Comment:
also could become a new span
and you should give some space between the "," and the spans
```suggestion
		to_chat(user, span_warning("You swap your grip, and will now perform a dive finisher."))
```